homeassistant/components/russound_rnet/media_player.py

```diff
@@ -85,17 +86,22 @@ def __init__(self, hass, russ, sources, zone_id, extra):
         self._attr_name = extra["name"]
         self._russ = russ
         self._attr_source_list = sources
-        self._zone_id = zone_id
+        self._controller_id = math.ceil(zone_id / 6)
```

Comment:
In the old code, the controller id was the string `"1"`. Is it OK also with an int?

Reply:
It does seem to work with int but I'll update to string